crowdin/src/test/java/com/crowdin/platform/CrowdinConfigTest.kt

```diff
@@ -78,4 +79,31 @@ class CrowdinConfigTest {
             // exception expected
         }
     }
+
+    @Test
+    fun whenUpdateIntervalLess15Minutes_shouldUseDefault() {
+        // Given
+        val smallInterval = 10 * 60L
+
+        // When
+        val configWithSmallInterval = CrowdinConfig.Builder()
+            .withDistributionHash("distributionHash")
+            .withUpdateInterval(smallInterval)
+            .build()
+
+        // Then
+        Assert.assertTrue(configWithSmallInterval.updateInterval == RecurringManager.MIN_PERIODIC_INTERVAL_MILLIS)
+
+        // Given
+        val bigInterval = 20 * 60L
```

Comment:
It is separate test case. As we have method name `whenUpdateIntervalLess15Minutes_shouldUseDefault` let's extract this part into another test case.
It could be something like this:
`setCustomUpdateInterval or setCustomUpdateInterval_successTest()`